scripts/arangodb-docker-container-setup/docker-container-scripts/test-data/testData.json

```diff
@@ -455,59 +455,146 @@
             {
```

Comment:
Really cool to see the event histories come through with only the desired changes (one term became a prompt term) to the state.